[core]/es_extended/client/main.lua

```diff
@@ -713,3 +713,65 @@ for i=1, #DoNotUse do
 		print("[^1ERROR^7] YOU ARE USING A RESOURCE THAT WILL BREAK ^1ESX^7, PLEASE REMOVE ^5"..DoNotUse[i].."^7")
 	end
 end
+
+-- Marker Interaction System
+local DrawingTextUI = false
+CreateThread(function()
+	while not ESX.PlayerLoaded do
+		Wait(0)
+	end
```

Comment:
why not use esx:playerLoaded ?? realy?

Reply:
> Good idea, I support it. However, I think the marker could be a class, similar to xPlayer, only xMarker with functions and events: playerHit, playerLeave etc

more memory usage, and not really needed :)

Reply:
> > Good idea, I support it. However, I think the marker could be a class, similar to xPlayer, only xMarker with functions and events: playerHit, playerLeave etc
> 
> more memory usage, and not really needed :)

better syntax, cleaner codes..
I'm looking forward to other people's opinions.

Reply:
I believe esx:playerLoaded would be a better approach especially in moving towards having an actual multicharacter system that gives you the ability to log out of your character and log back in with another one...